packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -76,7 +66,9 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
           {props.showErrorBar ? <ErrorBar {...errorBarProps} /> : <></>}
           <MessageThread
             {...messageThreadProps}
-            onRenderAvatar={onRenderMessageAvatar}
+            onRenderAvatar={(userId, options) => (
```

Comment:
Memoization here might be quite important: The message thread props are extensive, so MessageThread might be rerendered often. Also, there are probably many instances of each avatar (for each message), so it's worth avoiding rerendering them.

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/9782747/130145977-fd6d1399-3cd2-45d3-a878-de1f01b9a453.png)